content/200-orm/200-prisma-client/150-using-raw-sql/index.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Write your own SQL'
+metaTitle: 'Write Your Own SQL in Prisma Client'
+metaDescription: 'Learn how to use raw SQL queries in Prisma Client.'
+---
+
+While the Prisma Client API aims to make all your database queries intuitive, type-safe, and convenient, there may still be situations where raw SQL is the best tool for the job.
+
+This may happen for several reasons, such as the need to optimize the performance of a specific query or because your data requirements can't be fully expressed by Prisma Client's query API.
+
+In most cases, [TypedSQL](#writing-type-safe-queries-with-prisma-client-and-typedsql) will allow you to express your query in SQL while still leveraging Prisma Client's excellent user experience. However, since TypedSQL is statically typed, there are some cases (such as dynamically generated `WHERE` clauses) that it can't represent. For these cases, you will need to use [`$queryRaw`](/orm/prisma-client/using-raw-sql/raw-queries#queryraw) or [`$executeRaw`](/orm/prisma-client/using-raw-sql/raw-queries#executeraw) or their unsafe counterparts.
```

Comment:
```suggestion
While the Prisma Client API aims to make all your database queries intuitive, type-safe, and convenient, there may still be situations where raw SQL is the best tool for the job.

This can happen for various reasons, such as the need to optimize the performance of a specific query or because your data requirements can't be fully expressed by Prisma Client's query API.

In most cases, [TypedSQL](#writing-type-safe-queries-with-prisma-client-and-typedsql) allows you to express your query in SQL while still benefiting from Prisma Client's excellent user experience. However, since TypedSQL is statically typed, it may not handle certain scenarios, such as dynamically generated `WHERE` clauses. In these cases, you will need to use [`$queryRaw`](/orm/prisma-client/using-raw-sql/raw-queries#queryraw) or [`$executeRaw`](/orm/prisma-client/using-raw-sql/raw-queries#executeraw), or their unsafe counterparts.
```